ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/AddMetadataTest.java

```diff
@@ -71,178 +63,4 @@ public void testOutput() {
 
     pipeline.run();
   }
-
```

Comment:
everything after here was moved without changes to [ingestion-core/src/test/java/com/mozilla/telemetry/ingestion/core/transform/AddMetadataTest.java](https://github.com/mozilla/gcp-ingestion/pull/1330/files#diff-4175c5b7051ee412b8d6a4a0b7c0b342) below